graphql-dgs-platform/build.gradle.kts

```diff
@@ -46,10 +46,17 @@ dependencies {
     // For more information at https://docs.gradle.org/current/userguide/rich_versions.html
     constraints {
         api("com.graphql-java:graphql-java") {
-            version { require(Versions.GRAPHQL_JAVA) }
```

Comment:
Why get rid of the constant?

Reply:
The version is no explicitly set in the BOM. IMO it is easier to visually identify the versions, specially if there is correlation between two artifacts. In the past there were several places where the version was required making the constant useful, now there is only one.

Reply:
@paulbakker reverted the change with the constants. I also opened #243 to remove the rxjava recommendation.